calculus.tex

```diff
@@ -4,21 +4,25 @@
 
 Derivatives can be computed with the \verb|diff| function.
 \begin{verbatim}
+>>> x = symbols("x")
```

Comment:
This was needed because `x` is changed at the middle. `x = 1 / (sin(pi/5)+sin(2*pi/5)+sin(3*pi/5)+sin(4*pi/5))**2`
There's also a `x = Symbol('x', positive=True)` statement.